security-privacy-questionnaire.md

```diff
@@ -50,7 +49,7 @@ No.
 [GETUSERMEDIA](https://www.w3.org/TR/mediacapture-streams/) might expose deviceID. This specification does not expose anything extra.
 
 ### 13. How does this specification distinguish between behavior in first-party and third-party contexts?
-Capabilities in the ImageCapture as an extension of GetUserMedia, are exposed only to a top-level browsing secure context. In addition, the page must be visible when the website updates camera pan, tilt, and zoom with applyConstraints(), otherwise it fails with SecurityError.  Therefore ImageCapture can’t be used by third-party resources.
+Capabilities in the ImageCapture as an extension of GetUserMedia, are exposed only to a top-level browsing secure context. In addition, the page must be visible when the website updates camera pan, tilt, and zoom with applyConstraints(), otherwise it fails with SecurityError. Therefore ImageCapture can’t be used by third-party resources.
```

Comment:
> Therefore ImageCapture can’t be used by third-party resources.

Can we fix this existing statement? I think it's more correct to say that third-party access to `getUserMedia` is governed by the `"camera"` and `"microphone"` [permissions policies](https://w3c.github.io/mediacapture-main/getusermedia.html#permissions-policy-integration).